src/libstd/sys/windows/memchr.rs

```diff
@@ -12,4 +12,4 @@
 // Copyright 2015 Andrew Gallant, bluss and Nicolas Koch
 
 // Fallback memchr is fastest on windows
```

Comment:
Side note, this statement (Fallback memchr is fastest on windows) can be revisited by someone with the right development environment.